src/content/docs/usage/workspaces.mdx

```diff
@@ -193,148 +194,11 @@ Set the Project Builder at Workspace creation time using command-line
 flags. Refer to **[Builders](/docs/usage/builders)** for more information.
 </Aside>
 
-## Create a Workspace with Multiple Projects
```

Comment:
Can't put the comment above since it's unchanged, but please update the creation logs. They're different now since there are no more projects

Also, I would remove the TC selection stuff. It just bloats this section. We can just say (for example) `If you don't have a default target set (tip: daytona target set-default), you will be prompted to select a target for that workspace`